buildpacks/python/Makefile

```diff
@@ -168,6 +168,7 @@ buildpacks.python.tests.sources := \
 # Targets
 #########
 
+# buildpacks.python.images.local: invokers.python
```

Comment:
Did you mean?
```suggestion
buildpacks.python.images.local: invokers.python
```

Reply:
Good catch, thanks! That was leftover from some local debugging...